source/testnet/guides/get-started.rst

```diff
@@ -0,0 +1,130 @@
+
+.. _Set up a passcode and biometrics: #set-up-a-passcode-and-biometrics
+.. _Request your initial account and identity: #request-your-initial-account-and-identity
+.. _`the previous chapter`: get-the-app.html
+.. _Discord: https://discord.gg/xWmQ5tp
+
+
+=======================================
+Concordium ID: Get started with the app
+=======================================
+
+.. contents::
+   :local:
+   :backlinks: none
+
+Before following this guide you should have finished installing Concordium ID, as described in `the previous chapter`_.
+
+Set up a passcode and biometrics
+================================
+
+When you open the Concordium ID app for the first time, you will be greeted by a flow
+that will help you setup a passcode and biometric authentication, create an :ref:`glossary-initial-account`,
+and it will also guide you through getting an :ref:`glossary-identity-provider`. It is important to notice that the
+initial account will be submitted to the chain by the identity provider, but after your identity
+is created you will be able to make accounts yourself, that will be unknown to the identity provider.
```

Comment:
This paragraph jumps into warning you about initial accounts without explaining what they are.
Since this is an introductory guide I think it should provide a bit of context.

Reply:
I tried elaborating this in the newest commit.